rollup.config.js

```diff
@@ -1,27 +1,53 @@
-import commonjs from 'rollup-plugin-commonjs'
+import commonJs from 'rollup-plugin-commonjs'
```

Comment:
Why changing the case of a well-known name?

Reply:
No reason, except probably just being stuck on some thought process and mindlessly looking at other code will in the back of your head thinking about your problem and then changing this because commonJs would be the proper camelCase version of CommonJS even though it doesnt matter at all but hey I could probably do this to fix that other problem so ctrl+s and close. And then after what feels like 5mins you notice its dark outside and you start feeling to get hungry so you look for some food but you remember you need to go to the shops first because you have no food left but oh fuck its past 20.00 already and the shops are now closed...